projects/front-end-library/src/lib/components/selection-tile/angular/selection-tile.component.ts

```diff
@@ -1,5 +1,15 @@
 import { Component, ViewEncapsulation, OnInit, Input } from '@angular/core';
-import { Url } from 'url';
+import { ILink } from '../../link/angular/api.model';
+
+interface IFooter{
+    text: HTMLElement;
```

Comment:
```suggestion
    description: HTMLElement;
```
l'api text est problematique. Soyons plus précis/consistent.

Reply:
bien d'accord mais éviter d'attendre le 8e passage pour me le souligner.

Reply:
@bibifrost please resolve
SVP ne pas faire de suggestions quand le changement doit se faire à plusieurs endroits - c'est un coup pour casser le chmilblick

Reply:
Je te l'aurais dis avant, si je l'avait remarqué
Tu n'es pas obligé d'accepter la suggestion dans ce cas, c'est surtout pour bien comprendre mon commentaire.